examples/virtio/board/odroidc4/virtio.system

```diff
@@ -39,7 +39,7 @@
     </protection_domain>
 
     <protection_domain name="CLIENT_VMM-2" priority="100">
-        <program_image path="client_vmm_2.elf" />
+        <program_image path="client_vmm.elf" />
```

Comment:
The removal of separate client linux and roofs seems an unrelated change.

Reply:
I will move it into a separate PR. I still think client_vm_2 should be removed as it is identical to VM 1 and creates a lot of maintenance overhead

Reply:
Put it here https://github.com/au-ts/libvmm/pull/85